infoextract.js

```diff
@@ -124,51 +144,56 @@ module.exports = {
 		});
 	},
 
+	getManyPreviews(videoIds, qParamsV) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				const videoInfo = await this.getVideoInfoYoutube(videoIds);
+				resolve(videoIds.map(id => videoInfo[id])
+						   	.filter(info => info !== undefined) //remove deleted videos
+						   	.map(info => {
+									if (qParamsV && qParamsV === info.id) {
+										info.highlight = true;
```

Comment:
I don't think `highlight` should be set in here, it should be done in `getAddPreview`

Reply:
Why is that? I did it here so that it will run whenever the function is called, as it needs the video info to work correctly

Reply:
It makes the function less usable in other places, if we wanted to do that, because you need to pass in the query parameters from the url.

Reply:
It might be better to remove `highlight` altogether for now because it's currently not used.